package.json

```diff
@@ -10,7 +10,7 @@
     "coverage": "nyc report --reporter text-lcov | coveralls",
     "test": "nyc mocha",
     "lint": "eslint .",
-    "prepublish": "run-p test lint build",
+    "prepare": "run-p test lint build",
```

Comment:
Does this work with NPM5?

Reply:
Yes. https://docs.npmjs.com/misc/scripts#description . This preserves the old behavior (of working before publishing and with local installs). If you only want it working before publishing, then I can switch it to `prepublishOnly`, but I think it can't hurt to have local installs ensure it is built, linted, and passing tests...

Reply:
It will also work on npm4: https://resin.io/blog/safely-migrating-away-from-prepublish-with-npm-4/